packages/neutrino/src/neutrino.js

```diff
@@ -21,10 +21,17 @@ class Neutrino extends EventEmitter {
 
     this.config = new Config();
     this.options = merge(options, { root, source, output, tests, node_modules, entry });
+    this.useQueue = []
   }
 
   use(preset, options = {}) {
-    preset(this, options);
+    this.useQueue.push(() => preset(this, options));
+  }
+
+  load() {
+    return this.useQueue.reduce((result, task) => {
```

Comment:
Good catch. No need for assigning the result variable as I am already returning the result in reduce 👍